tests/plugins/pastefromwordimage/manual/pastewordwithimage.md

```diff
@@ -13,6 +13,9 @@
   2. [Online_and_offline_image.docx](../generated/_fixtures/Online_and_offline_image/Online_and_offline_image.docx)
   3. [Shapes_and_online_and_offline_image.docx](../generated/_fixtures/Shapes_and_online_and_offline_image/Shapes_and_online_and_offline_image.docx)
   4. [Wrapped_image.docx](../generated/_fixtures/Wrapped_image/Wrapped_image.docx)
+  5. [Image_alternative_text.docx](../generated/_fixtures/Image_alternative_text/Image_alternative_text.docx)
+  6. [Image_reflection.docx](../generated/_fixtures/Image_reflection/Image_reflection.docx)
+  7. [Image_rotation.docx](../generated/_fixtures/Image_rotation/Image_rotation.docx)
```

Comment:
When pasting rotated image it is pasted in its original form (not rotated). I'm not sure if it is a proper behaviour or not (as for previous `.docx` images are pasted exactly as they appear in the document). If this is an intended behaviour it should be explain that rotated images are pasted, but without rotation.